token.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <string>
+
+namespace dev {
+
+class Token {
```

Comment:
Do we really need this class?I would remove it.

Reply:
Removed